### DISCUSSION TITLE
redirect back to nuxt page from third party page can not trigger SSR rerender

### DISCUSSION THREAD

**Discussion by KahrLu** [nuxt/nuxt#17073] [Questions]
### Environment

any env

### Reproduction

any version

### Describe the bug

Hi @pi0,My project needs to jump with the front-end projects of other technology stacks. When I jump from another project page to the nuxt page or return to the nuxt page, SSR rerender cannot be triggered

**Comment by danielroe** [comment]
You can use an `<a>` tag to force hard SSR, and you can also do this programmatically within a route middleware if your use case is slightly more complex.

**Comment by KahrLu** [comment]
![image](https://user-images.githubusercontent.com/76502123/184671957-cbfedea8-8324-492d-abe5-18bae5ee054c.png)
I use the above code to hard refresh the page, but the experience is not very good.
